govtool/frontend/src/components/organisms/DashboardCards/DRepDashboardCard.tsx

```diff
@@ -120,7 +120,7 @@ export const DRepDashboardCard = ({
               dataTestId: "register-learn-more-button",
               onClick: () =>
                 openInNewTab(
-                  "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/dreps/retire-as-a-drep",
+                  "https://docs.gov.tools/using-govtool/govtool-functions/dreps/register-as-a-drep",
                 ),
             }
           : learnMoreButton,
```

Comment:
The previous link was for retirement, is the change intentional? If it co, then it's basically the same as `learnMoreButton`

Reply:
It should be retire - thanks for spotting this! :saluting_face: